lib/plugins/aws/provider.js

```diff
@@ -521,7 +521,7 @@ class AwsProvider {
           awsResourceTags: {
             type: 'object',
             patternProperties: {
-              '^(?!aws:)[\\w./=+:-_\\x20]{1,128}$': {
+              '^(?!aws:)[\\w\\./=+:\\-_\\x20]{1,128}$': {
```

Comment:
There's no need to escape `.`

Reply:
I figured it would be clearer to escape it, but okay, I'll revert